moped-api/users/users.py

```diff
@@ -1,31 +1,34 @@
-import os, boto3
+import boto3
+
 from botocore.exceptions import ClientError
-from flask import Blueprint, jsonify, abort
+from flask import Blueprint, jsonify, abort, Response
```

Comment:
Cool, TIL. This is a much better option for the return type hints in this blueprint.